#Scenes/MapUI.tscn

```diff
@@ -20,18 +21,21 @@ grow_horizontal = 2
 grow_vertical = 2
 color = Color(0, 0, 0, 0.309804)
 
-[node name="ScrollContainer" type="ScrollContainer" parent="."]
+[node name="SmoothScrollContainer" type="ScrollContainer" parent="."]
 layout_mode = 0
-offset_left = 260.0
+offset_left = 223.0
 offset_top = 60.0
-offset_right = 1059.0
+offset_right = 1123.0
 offset_bottom = 660.0
-follow_focus = true
+script = ExtResource("2_2gata")
+speed = 2.0
+allow_overdragging = false
 
-[node name="RoomContainer" type="ColorRect" parent="ScrollContainer"]
+[node name="RoomContainer" type="ColorRect" parent="SmoothScrollContainer"]
 clip_contents = true
-custom_minimum_size = Vector2(799, 600)
+custom_minimum_size = Vector2(900, 600)
```

Comment:
can the minimum size be deduced depending on the size of the map needed ? based on the number of rooms maybe ? That would allow an adaptable size

Reply:
Ya i should be able to. I do so with the height, can do it for the width.